ovos_PHAL_plugin_network_manager/__init__.py

```diff
@@ -170,10 +172,25 @@ def handle_network_connect_request(self, message):
             # TODO: Implement dbus backend
             LOG.info("Connecting to network using dbus backend")
         if self.backend == "nmcli":
-            cmd = ["nmcli", "device", "wifi", "connect", network_name]
-            if secret_phrase:
-                cmd.extend(["password", secret_phrase])
-            connection_process = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+            connection_process = subprocess.Popen(
+                ["nmcli", "device", "wifi", "connect", network_name, "password", secret_phrase], stdout=subprocess.PIPE)
```

Comment:
I don't think this is right for an open network; the `password` param should be omitted if `secret_phrase` is an empty string (or None)

Reply:
This has been fixed in the latest commit and also in the GUI PHAL plugin open PR